.eslintrc.js

```diff
@@ -1,16 +1,31 @@
-const { node: baseConfig, overwritePresets } = require('@sumup/foundry/eslint');
-
-const customConfig = {
-  overrides: [
-    {
-      files: ['files/**/*.js'],
-      rules: {
-        'notice/notice': 'off',
-        'import/no-unresolved': 'off',
-        'no-unused-vars': 'warn'
-      }
-    }
-  ]
-};
-
-module.exports = overwritePresets(baseConfig, customConfig);
+module.exports = require('@sumup/foundry/eslint')(
+  {
+    language: 'JavaScript',
+    environments: ['Node'],
+    openSource: true,
+  },
+  {
+    env: {
+      node: true,
+    },
+    overrides: [
+      {
+        files: ['files/**/*.js'],
+        env: {
+          browser: true,
+        },
+        rules: {
+          'notice/notice': 'off',
+          'import/no-unresolved': 'off',
+          'no-unused-vars': 'warn',
+        },
+      },
+      {
+        files: ['files/**/*.spec.js'],
+        env: {
+          jest: true,
+        },
+      },
+    ],
+  },
```

Comment:
None of this should be required anymore if you're using Foundry.

Reply:
If I don't make this happen, lint was complaning 🤷‍♂️ 
the override in `files/**/*.js` was here before

Reply:
That's what I mean, those overrides are obsolete now 😉

Reply:
But without the overrides, the lint is complaining about the jest globals not being defined 🤔

Reply:
You're missing `frameworks: ['Jest']` in the config

Reply:
https://github.com/sumup-oss/foundry#-lint

Reply:
AH, got it